core/src/main/scala/org/ensime/indexer/FileWatchers.scala

```diff
@@ -155,3 +167,116 @@ private class ApachePollingFileWatcher(
     fm.stop()
   }
 }
+
+object JarJava7Watcher {
+  lazy val jarWatcher = new Java7Watcher()
```

Comment:
I'm not too keen on this as an `object` as it introduces global, mutable, state.